nevergrad/benchmark/experiments.py

```diff
@@ -1129,6 +1131,229 @@ def rocket(seed: tp.Optional[int] = None, seq: bool = False) -> tp.Iterator[Expe
+                        if not xp.is_incoherent:
+                            yield xp
+
+
+@registry.register
+def multi_ng_full_gym(seed: tp.Optional[int] = None) -> tp.Iterator[Experiment]:
+    return ng_full_gym(seed, multi=True)
+
+
+@registry.register
+def conformant_ng_full_gym(seed: tp.Optional[int] = None) -> tp.Iterator[Experiment]:
+    """Counterpart of gym_multi with fixed, predetermined actions for each time step."""
+    return ng_full_gym(seed, conformant=True)
+
+
+@registry.register
+def ng_gym(seed: tp.Optional[int] = None) -> tp.Iterator[Experiment]:
+    """Counterpart of gym_multi with a specific list of problems."""
+    return ng_full_gym(seed, ng_gym=True)
+
+
+@registry.register
+def ng_stacking_gym(seed: tp.Optional[int] = None) -> tp.Iterator[Experiment]:
+    """Counterpart of gym_multi with a specific list of problems."""
+    return ng_full_gym(seed, ng_gym=True, memory=True)
+
+
+@registry.register
+def memory_gym_multi(seed: tp.Optional[int] = None) -> tp.Iterator[Experiment]:
+    """Counterpart of gym_multi with a recurrent net as a memory."""
+    return ng_full_gym(seed, big=False, memory=True)
+
+
+@registry.register
+def memory_big_gym_multi(seed: tp.Optional[int] = None) -> tp.Iterator[Experiment]:
+    """Counterpart of gym_multi with bigger nets and a recurrent net as a memory."""
+    return ng_full_gym(seed, big=True, memory=True)
+
+
+@registry.register
+def big_gym_multi(seed: tp.Optional[int] = None) -> tp.Iterator[Experiment]:
+    """Counterpart of gym_multi with bigger nets."""
+    return ng_full_gym(seed, big=True)
+
+
+@registry.register
+def deterministic_gym_multi(seed: tp.Optional[int] = None) -> tp.Iterator[Experiment]:
+    """Counterpart of gym_multi with fixed seeds (so that the problem becomes deterministic)."""
+    return ng_full_gym(seed, randomized=False)
+
+
+@registry.register
+def gym_multifid_anm(seed: tp.Optional[int] = None) -> tp.Iterator[Experiment]:
+    """Gym simulator for Active Network Management."""
+
+    func = GymMulti("multifidLANM")
+    seedg = create_seed_generator(seed)
+    optims = get_optimizers("basics", "progressive", "splitters", "baselines", seed=next(seedg))
+    for budget in [25, 50, 100, 200, 400, 800, 1600, 3200, 6400, 12800, 25600, 51200]:
+        for num_workers in [1, 30]:
+            if num_workers < budget:
+                for algo in optims:
+                    xp = Experiment(func, algo, budget, num_workers=num_workers, seed=next(seedg))
+                    if not xp.is_incoherent:
+                        yield xp
+
+
+@registry.register
+def gym_anm(
+    seed: tp.Optional[int] = None,
+    specific_problem: str = "LANM",
+    conformant: bool = False,
+    pb_index: int = -1,
+) -> tp.Iterator[Experiment]:
+    """Gym simulator for Active Network Management or other pb."""
+    if "directcompilergym" in specific_problem:
+        assert pb_index >= 0
+        funcs = [CompilerGym(pb_index)]
```

Comment:
Maybe CompilerGym should be renamed to DirectCompilerGym since it only handles the direct version ?

Reply:
Yes.